### DISCUSSION TITLE
Unable to make request to my Node.js backend server

### DISCUSSION THREAD

**Discussion by kartikvxceed** [GrapesJS/grapesjs#5233] [General]
grapesjs.init({
            blockManager: {

            },
            container: '#gjs',
            height: '900px',
            width: '100%',
            plugins: ['gjs-preset-webpage'],
            storageManager: {
                type: 'remote',
                stepsBeforeSave: 1,
                contentTypeJson: true,
                storeComponents: true,
                storeStyles: true,
                storeHtml: true,
                storeCss: true,
                id: 'my-',
                urlLoad: projectEndpoint,
                urlStore: projectEndpoint,
                headers: {
                    'Content-Type': 'application/json'
                },
                onStore: data => ({ id: '1', data }),
                onLoad: result => result.data,
            },
            pluginsOpts: {
                'grapesjs-preset-webpage': {
                    "gjs-blocks-basic": {
                        stylePrefix: '',
                        flexGrid: 1
                    },
                    blocksBasicOpts: {
                        blocks: ['column1', 'column2', 'column3', 'column3-7', 'text', 'link', 'image', 'video'],
                        flexGrid: 1,
                    },
                    blocks: ['link-block', 'quote', 'text-basic'],
                },
            }
        })

This is my grapes.init in a vue2 app used in mounted life cycle method. Every time I refresh the page, the request goes to my current front-end route (localhost:8080/builder) and not to the Node.js backend route (localhost:5000/pages)

**Comment by artf** [comment]
Same as [here](https://github.com/GrapesJS/grapesjs/discussions/5216#discussioncomment-6401556)
